numpy/core/numeric.py

```diff
@@ -376,13 +376,13 @@ def full_like(a, fill_value, dtype=None, order='K', subok=True):
     >>> np.full_like(x, 0.1)
     array([0, 0, 0, 0, 0, 0])
     >>> np.full_like(x, 0.1, dtype=np.double)
-    array([ 0.1,  0.1,  0.1,  0.1,  0.1,  0.1])
+    array([0.1, 0.1, 0.1, 0.1, 0.1, 0.1])
     >>> np.full_like(x, np.nan, dtype=np.double)
-    array([ nan,  nan,  nan,  nan,  nan,  nan])
+    array([nan,  nan,  nan,  nan,  nan,  nan])
```

Comment:
This spacing looks wrong to me. I thought we killed these extra spaces

Reply:
That's what the code produces

Reply:
I'll remove the intervening spaces locally -- the important part of the diff was removing the leading space, but that may have been left over from the doctest-strict changes. The intervening spaces are probably just fine for refguide so that passes through

Reply:
pushed the space clean-up, marking as resolved